### DISCUSSION TITLE
How to build android?

### DISCUSSION THREAD

**Discussion by jiangjiali** [mamedev/discussions#86] [Q&A]
How to build android?
Do you have a detailed tutorial.

**Comment by jiangjiali** [comment]
https://github.com/jiangjiali/mame0261/actions/runs/7550566188/job/20556408298

**Comment by galibert** [comment]
type_traits is necessary and part of the C++11 standard.  So you need to find a way to enable C++11 in your sdk.

Honestly nobody in the core team ever tries to build for android, so we can't provide more specific help.
